### PR TITLE
py2<->py3 compatible

### PR DESCRIPTION
**Pull request by vatsa287**
Changes made to storage_add.py , install.py , main.py. Modules utils.py and __init__.py complied in both versions.

### CODE REVIEW

**Comment by amarts** [comment]
You need to do a `git commit -s -a --amend` and then do a `git push --force` again to pass DCO tests.

**Comment by amarts** [comment]
Also run `make pylint` once to pass the tests

```
cli/kubectl_kadalu/storage_add.py:4:37: C0303: Trailing whitespace (trailing-whitespace)
cli/kubectl_kadalu/storage_add.py:15:0: C0303: Trailing whitespace (trailing-whitespace)
```

**Comment by vatsa287** [comment]
``` 
git commit -s -a --amend -m "py2-3"
[py2-py3 cc6229e] py2-3
 Author: Amar Tumballi <amar@kadalu.io>
 Date: Tue Mar 24 10:09:39 2020 +0530
 4 files changed, 39 insertions(+), 5 deletions(-)
 create mode 100644 .github/stale.yml
vatsa@vatsa-VirtualBox:~/kadalu/cli/kubectl_kadalu$ make pylint
make: *** No rule to make target 'pylint'.  Stop.
```

**Comment by aravindavk** [comment]
> vatsa@vatsa-VirtualBox:~/kadalu/cli/kubectl_kadalu$ make pylint
> make: *** No rule to make target 'pylint'.  Stop.

Run `make pylint` from project root directory

File: cli/kubectl_kadalu/install.py [hunk 1]
```diff
@@ -1,10 +1,14 @@
...
-import sys
+from __future__ import print_function
 
-from kubectl_kadalu import utils
+import sys
 
+try:
+    import utils
+except ImportError:
+    from kubectl_kadalu import utils
```
Comment (aravindavk) [inline cli/kubectl_kadalu/install.py:11]: These should work in Python2 as well I think.

**Comment by aravindavk** [comment]
Try pylint of both versions.

```
cd cli
pylint2 --disable W0511,R0801,W0611 kubectl_kadalu
pylint3 --disable W0511,R0801,W0611 kubectl_kadalu
```

Try this with only `from __future__ import print_function` and try the above commands.

**Comment by vatsa287** [comment]
vatsa@vatsa-VirtualBox:~/kadalu$ sudo make pylint
No config file found, using default configuration
************* Module kadalulib
E: 35, 0: invalid syntax (<string>, line 35) (syntax-error)
Makefile:41: recipe for target 'pylint' failed
make: *** [pylint] Error 2

**Comment by vatsa287** [comment]
@aravindavk anna i removed try and except and from __future__ import print_function is present
also there was import error and used relative import 
```
vatsa@vatsa-VirtualBox:~/kadalu/cli$ pylint3 --disable W0511,R0801,W0611 kubectl_kadalu
No config file found, using default configuration
************* Module kubectl_kadalu.install
E: 49,15: Module 'utils' has no 'KUBECTL_CMD' member (no-member)
E: 51,15: Module 'utils' has no 'execute' member (no-member)
E: 55,11: Module 'utils' has no 'CommandError' member (no-member)
************* Module kubectl_kadalu.storage_add
E:133,15: Module 'utils' has no 'execute' member (no-member)
E:147,11: Module 'utils' has no 'CommandError' member (no-member)
E:235,15: Module 'utils' has no 'KUBECTL_CMD' member (no-member)
E:236,15: Module 'utils' has no 'execute' member (no-member)
E:243,11: Module 'utils' has no 'CommandError' member (no-member)

------------------------------------------------------------------
Your code has been rated at 7.93/10 (previous run: 9.43/10, -1.50)
```
```
vatsa@vatsa-VirtualBox:~/kadalu/cli$ pylint --disable W0511,R0801,W0611 kubectl_kadalu
No config file found, using default configuration
************* Module kubectl_kadalu.main
W: 14, 0: Relative import 'storage_add', should be 'kubectl_kadalu.storage_add' (relative-import)
W: 15, 0: Relative import 'install', should be 'kubectl_kadalu.install' (relative-import)
************* Module kubectl_kadalu.utils
C: 10, 0: Old-style class defined. (old-style-class)
C: 12, 4: Argument name "rc" doesn't conform to snake_case naming style (invalid-name)
E: 21, 8: Missing argument to super() (missing-super-argument)
************* Module kubectl_kadalu.install
W:  9, 0: Relative import 'utils', should be 'kubectl_kadalu.utils' (relative-import)
************* Module kubectl_kadalu.storage_add
W: 13, 0: Relative import 'utils', should be 'kubectl_kadalu.utils' (relative-import)

-----------------------------------
Your code has been rated at 9.43/10

```

**Comment by amarts** [comment]
@vatsa287 when you get a chance, can you rebase and send the patch?

**Comment by vatsa287** [comment]
@amarts yes anna . sure!

**Pull request closed without merging**